docs/pages/application-access/okta/scim-only.mdx

```diff
@@ -0,0 +1,166 @@
+user already has a valid temporary Teleport SAML user account (i.e. they have
+logged into the cluster via SAML SSO before SCIM provisioning was enabled), the
+temporary account will automatically be adopted by the SAML integration and
+promoted to a long-lived SCIM-managed account.
+
+If a Teleport user with the same name as the Okta user already exists (excluding the temporary SAML user case above), provisioning that user will fail. Provisioning errors will show in both the Okta Tasks page and on the affected user in the Okta application’s assignments pane.
+
+<Admonition type="note">
+Currently none of the SCIM user profile traits are stored in the Teleport user, although this may change in future. 
+</Admonition>
+
+When a user is unassigned from the Okta App, or is disabled by the Okta Admin, Teleport will immediately delete the user in question, and create a lock that will both immediately terminate any existing sessions and prevent that user from re-using any credentials that have already been issued. This lock will be automatically revoked if the user is subsequently re-provisioned via SCIM, otherwise the lock is permanent and will have to be explicitly deleted to remove.
+
+<Admonition type="warning">
+Okta does not send SCIM updates to Teleport when a user is merely suspended. Even though Okta will prevent a suspended user from logging back into the cluster, any existing sessions will not be terminated, and any pre-issued credentials will be valid for their normal lifetimes.
+</Admonition>
+
+## What you need
+ * Enterprise Teleport Dev Build `v15.3.2-dev.scim.1`, available via 
+   * AMD64 linux: https://cdn.teleport.dev/teleport-ent-v15.3.2-dev.scim.1-linux-amd64-bin.tar.gz 
+   * Arm64 linux: https://cdn.teleport.dev/teleport-ent-v15.3.2-dev.scim.1-linux-arm64-bin.tar.gz
+   * Multiarch Docker image (with debug tools): public.ecr.aws/gravitational/teleport-ent-distroless-debug:15.3.2-dev.scim.1
+ * A Teleport cluster with a working SAML Connector to your Okta organization
+ * A securely-generated bearer token
+
+## Installing Teleport
+
+If installing from one of the above tarballs onto a self-hosted system, first 
+back up your existing binaries then unpack the linked tarball and move the 
+contained binaries into place.   
+
+## Installing the SAML Connector
+If you created an Okta SAML connector with the Hosted Okta Integration enrollment
+flow, it will have created a SAML Connector called okta-integration that assigns
+users the okta-requester to the Okta group Everyone.
+
+If you don’t already have a working SAML Connector, see the 
+[Teleport Authentication with Okta](../../access-controls/sso/okta) guide. You
+can skip the Okta Group and role creation parts, as they’re not necessary for a
+SCIM deployment
+
+The resulting Teleport SAML connector resource should look roughly like the 
+example below. Note the role mapping that grants the requester role to the
+`Everyone` group in Okta.
```

Comment:
This phrasing is slightly confusing - is it a requirement to have the role mapping for "everyone" setup, or just an example and users can use any existing connector they have?